File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashMap<Integer, Integer> mapping = new HashMap<>();
        for (int i = 0; i < nums.length; i++) {
            if (!mapping.containsKey(nums[i])) {
                mapping.put(nums[i], 1);
            }
            else {
                return true;
            }
        }
        return false;

    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        mapping = defaultdict(int)
        for num in nums:
            mapping[num]+=1
        
        elems = list(mapping.values())
        returnList = [0] * k
        for i in range(k):
            returnList[i] = next((key for key, value in mapping.items() if value == max(elems)), None)
            elems.remove(max(elems))

        return returnList



